implementacija/app/Views/register.php

```diff
@@ -8,51 +8,58 @@
 <div class="reg">
     <div class="container-reg">
         <div class="wrap-reg">
-            <form>
+            <form method="POST" action="<?= base_url('GuestController/OPregister')?>">
                 <span class="reg-title p-b-34 p-t-27">
```

Comment:
Jel ove klase (p-b-34 p-t-27) uopste rade nesto?

Reply:
Nisam sigurna, to je Filipov html/css iz prototipa